libcontainer/configs/validate/validator.go

```diff
@@ -7,7 +7,7 @@ import (
 	"strings"
 
 	"github.com/opencontainers/runc/libcontainer/configs"
-	"github.com/opencontainers/runc/libcontainer/selinux"
+	selinux "github.com/opencontainers/selinux/go-selinux"
```

Comment:
@runcom says there is no need to use 

`selinux` "github.com/opencontainers/selinux/go-selinux"

Here, it will work fine without it.